client/src/helpers/getDOIUrl.js

```diff
@@ -0,0 +1,5 @@
+// takes a DOI and returns the url
+
+export const getDOIUrl = (doi) => `http://dx.doi.org/${doi}`
```

Comment:
single file per function?

Reply:
Yeah I was thinking it would be easier to find and edit. It will be by functionality so nothing preventing multiple functions per file. Basically just don't want a long file of unrelated functions.